code/modules/clothing/rogueclothes/wrists.dm

```diff
@@ -265,3 +265,32 @@
 	pickup_sound = 'sound/foley/equip/equip_armor_chain.ogg'
 	equip_sound = 'sound/foley/equip/equip_armor_chain.ogg'
 	smeltresult = null
+
+//
+
+/obj/item/clothing/wrists/roguetown/bracers/matthios
```

Comment:
Okay. I should've made one review but it's all three of the Matthios armors, actually.